src/app/components/cardsSection/cardsSection.tsx

```diff
@@ -35,6 +58,9 @@ export default function CardsSection({ label }: CardsSectionProps) {
         <Card key={card.id}>
           <header className={styles.cardHeader}>{card.title}</header>
           <p className={styles.cardContent}>{card.content}</p>
+          <Link href={card.href} passHref>
+            <Button buttonText={card.buttonText} showIcon={true} />
```

Comment:
nit and non-blocking: Instead of explicitly setting `showIcon={true}`, we can simply use `showIcon` as a prop, and it will be considered true by default.